nixos/modules/services/backup/zfs-replication.nix

```diff
@@ -73,7 +80,7 @@ in {
         "https://github.com/alunduil/zfs-replicate"
       ];
       restartIfChanged = false;
-      serviceConfig.ExecStart = "${pkgs.zfs-replicate}/bin/zfs-replicate${recursive} -l ${escapeShellArg cfg.username} -i ${escapeShellArg cfg.identityFilePath}${followDelete} ${escapeShellArg cfg.host} ${escapeShellArg cfg.remoteFilesystem} ${escapeShellArg cfg.localFilesystem}";
+      serviceConfig.ExecStart = "${pkgs.zfs-replicate}/bin/zfs-replicate${recursive} -p ${escapeShellArg toString cfg.port} -l ${escapeShellArg cfg.username} -i ${escapeShellArg cfg.identityFilePath}${followDelete} ${escapeShellArg cfg.host} ${escapeShellArg cfg.remoteFilesystem} ${escapeShellArg cfg.localFilesystem}";
```

Comment:
What will happen if the remote host is configured via ssh's config to be a specific port, will the `-p` here override that, possibly breaking those users?

Reply:
@grahamc I'm not sure I completely follow.  Are you asking if the /root/.ssh/config includes a Port directive for this host will that be overridden in this execution?  The answer is yes and it currently is since zfs-replicate defaults the port argument to 22.

Let me know if I'm misunderstanding the question though.

Reply:
@grahamc is there anything I can do to help get this landed?